src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -370,13 +370,27 @@ private void RefreshBackupFileList(IEnumerable<string> filePaths)
             {
                 var extension = Path.GetExtension(filePath).ToUpper();
                 var caption = Path.GetFileNameWithoutExtension(filePath);
-                files.Add(new StartPageListItem(caption)
+                if (extension.IndexOf(".") == 0)
                 {
-                    ContextData = filePath,
-                    ToolTip = filePath,
-                    SubScript = extension.Substring(1), // Skipping the 'dot'
-                    ClickAction = StartPageListItem.Action.FilePath
-                });
+                    files.Add(new StartPageListItem(caption)
```

Comment:
Seems like it would be much simpler to just do this between 371 and 372 and not duplicate all that code:
```
if(extension.IndexOf(".") == 0)
{
    extension = extension.Substring(1);
}
```